stablehlo/reference/Element.cpp

```diff
@@ -607,6 +601,203 @@ Element ceil(const Element &el) {
   return Element(el.getType(), val);
 }
 
+Element convert(Type type, bool value) {
+  if (isSupportedBooleanType(type)) return Element(type, value);
+  if (isSupportedIntegerType(type))
+    return Element(type, APInt(type.getIntOrFloatBitWidth(), value ? 1 : 0,
+                               type.cast<IntegerType>().isSigned()));
+  if (isSupportedFloatType(type)) {
+    bool roundingErr;
+    auto floatVal = APFloat(value ? 1.0 : 0.0);
```

Comment:
Can we create floatVal using the right semantics in the first place? This way we won't need to call convert afterwards? Same below.

Reply:
APFloat does not have a constructor to create one with the correct semantics, so convert in one way or another is inevitable. For example, we could use the `convertFromAPInt` from the APFloat library, but the current code is more concise.